app/controllers/application_controller.rb

```diff
@@ -0,0 +1,2 @@
+class ApplicationController < ActionController::Base
+end
```

Comment:
- [ ] It will be great to add a method called `current_user` that returns the first user in the application. This is a major requirement for this project. Kindly check the project description below ⬇️ 
![Screenshot 2023-05-13 at 19 33 35](https://github.com/Mukumbuta/My_Bloging_App/assets/32361963/9fe69e44-79cc-471b-b0aa-ff4e8cbb912c)